### PR TITLE
fix: enable `InspectorAddon` on release builds

### PR DESCRIPTION
**Pull request by dominic-cot**
By passing `enabled` to the `Inspector` widget we ensure that the inspector is properly toggled for release builds. See issue below for further details.

### List of issues which are fixed by the PR
- https://github.com/widgetbook/widgetbook/issues/1085

### Screenshots
For the following screenshots I ran `flutter build web` and then ran the build with `dhhtpd`.
First is before change, and second is with my change, where you can see the `Inspector` is now shown.

image

image

### Checklist

- [x] I signed the [CLA].
- [x] I listed at least one issue that this PR fixes in the description above.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making].
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on [Discord].

[CLA]: https://docs.google.com/forms/d/e/1FAIpQLScuRfjUzENsLsmQgqZlGLxMKbFi7zuXoPARyXytoyQrq7ntUw/viewform
[Discord]: https://discord.com/invite/zT4AMStAJA

### CODE REVIEW

File: packages/widgetbook/lib/src/addons/inspector_addon/inspector_addon.dart [hunk 1]
```diff
@@ -34,7 +34,10 @@ class InspectorAddon extends WidgetbookAddon<bool> {
     return setting
```
Comment (YoussefRaafatNasry) [inline packages/widgetbook/lib/src/addons/inspector_addon/inspector_addon.dart:34]: I don't think the conditional statement make sense after passing the `isEnabled` parameter.
We can remove both ternary operator and the scaffold as follows:

```suggestion
    return Inspector(
      isEnabled: setting,
      child: child,
    );
```

**Review by YoussefRaafatNasry — CHANGES_REQUESTED** [review]

File: packages/widgetbook/lib/src/addons/inspector_addon/inspector_addon.dart (hunk 1 above)
Comment (dominic-cot) [reply packages/widgetbook/lib/src/addons/inspector_addon/inspector_addon.dart:34]: That's fair enough, I assumed there was a reason that originally the addon was wrapping the `Inspector` in a `Scaffold`.

Comment (dominic-cot) [reply packages/widgetbook/lib/src/addons/inspector_addon/inspector_addon.dart:34]: Would also fix that the addon currently changes the background color which probably isn't desired behaviour.

Comment (YoussefRaafatNasry) [reply packages/widgetbook/lib/src/addons/inspector_addon/inspector_addon.dart:34]: It probably just skipped the review during Hacktoberfest, but no reason of having it there.

Comment (dominic-cot) [reply packages/widgetbook/lib/src/addons/inspector_addon/inspector_addon.dart]: Ah I'll revert that change then 😁

**Review by YoussefRaafatNasry — APPROVED** [review]
I have updated the PR with the changes to get it merged before the ongoing release.
Thanks a lot for you contribution! 💙

**Pull request merged**